### ISSUE TITLE
DPL-740-1 - As developers we need to modify the tests to use the instrument type configuration to make it more flexible

### ISSUE THREAD

**Issue by stevieing** [sanger/traction-service#1041]
**User story**
As developers we need to modify the tests to use the instrument type configuration to make it more flexible, Currently the factories are tied to having 2 plates for Revio which means they are inflexible and difficult to maintain

**Who are the primary contacts for this story**
steve

**Who is the nominated tester for UAT**
PSD

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] Modify the pacbio run fatory to be configuration driven so that it could be revio, sequel IIe and have any number of wells and positions
- [ ] Modify the pacbio run tests so they can use 1 or 2 plates for Revio

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

References
This story has a non-blocking relationship with:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**

**Comment by SujitDey2022** [comment]
@stevieing seems like the story numbers are duplicated with the one here - https://github.com/sanger/limber/issues/702

**Comment by stevieing** [comment]
Closed as no longer relevant
